atlassian_jwt_auth/signer.py

```diff
@@ -25,6 +28,25 @@ def __init__(self, issuer, private_key_retriever, **kwargs):
             raise ValueError("lifetime, '%s',exceeds the allowed 1 hour max" %
                              (self.lifetime))
 
+    def _obtain_private_key(self, key_identifier, private_key_pem):
```

Comment:
Nit: `obtain` is a little too fancy for me. Why not `get`?

Reply:
I prefer `obtain` over `get` in this case because it isn't a simple access.